tools/challenge-helper-scripts/repair-meta.ts

```diff
@@ -0,0 +1,30 @@
+import { readdir } from 'fs/promises';
+import { join } from 'path';
+
+import * as matter from 'gray-matter';
+
+import { getProjectPath } from './helpers/get-project-info';
+import { getMetaData, updateMetaData } from './helpers/project-metadata';
+
+const sortByStepNum = (a: string, b: string) =>
+  parseInt(a.split('-')[1]) - parseInt(b.split('-')[1]);
```

Comment:
Everything inside of me is saying _"OPTIONAL CHAINING"_, otherwise, this does _technically_ work 👍